src/components/students/Students.tsx

```diff
@@ -0,0 +1,59 @@
+import { faUserPlus } from "@fortawesome/free-solid-svg-icons"
+import Navbar from "../navbar/Navbar"
+import SearchBar from "../SearchBar"
+import './Students.css'
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome"
+import { Link } from "react-router-dom"
+
+
+const students=[{id:1,fullname:"جميلة عامري",email:"jamilaamri@gmail.com",memoQuantity:"3 صفحات",disciplineMark:"+++"},
+    {id:2,fullname:"مريم بن سالم",email:"meriembensalem@gmail.com",memoQuantity:"2 أجزاء",disciplineMark:"++"},
+    {id:3,fullname:"أحمد السريدي",email:"ahmedsridi@gmail.com",memoQuantity:"5 صفحات",disciplineMark:"++"},
+    {id:4,fullname:"محسن بن قاسم",email:"mohsenbenkacem@gmail.com",memoQuantity:"3 أجزاء",disciplineMark:"+++"},
+]
+
+const Students = () => {
+  return (
+    <div>
```

Comment:
for the table we should use a table package, it's hard to use native table directly to manage filtering ect. 
https://www.ag-grid.com/